munch-match/src/components/resultsLayout.jsx

```diff
@@ -1,105 +1,157 @@
 //Tells react that this is an component
-import { Component } from "react";
+import {Component} from "react";
 import "../App.css"
 import ResultCard from "./resultCard"
-import { getResults } from "../services/MapsService.js"
+import {getResults} from "../services/MapsService.js"
 import styles from "./resultsLayout.module.css"
+import RangeSlider from 'react-range-slider-input';
+import 'react-range-slider-input/dist/style.css';
 
 //Creates the google map variable
-const google = window.google; 
+const google = window.google;
 
 //Creates the Button component
-class Results extends Component{
+class Results extends Component {
 
     state = {
         results: [],
-      };
-    
-      onBackClick = () => {
+        ratingRange: [0, 5],
+        priceRanges: {
+            1: true,
+            2: true,
+            3: true,
+        },
+    };
+
+    onBackClick = () => {
         this.props.onBackClick();
-      };
-    
-      componentDidMount() {
+    };
+
+    componentDidMount() {
         // Obtain user location when results are requested 
         this.fetchUserLocation().then((position) => {
-          this.fetchResults(position.coords.latitude, position.coords.longitude);
+            this.fetchResults(position.coords.latitude, position.coords.longitude);
         }).catch((err) => {
-          console.log("Error in retrieving user location")
-        }); 
-      }
+            console.log("Error in retrieving user location")
```

Comment:
A suggestion would be to consider more robust error message instead of doing a console log